mat/symmetric.go

```diff
@@ -483,12 +482,12 @@ func (s *SymDense) PowPSD(a Symmetric, pow float64) error {
 	var eigen EigenSym
 	ok := eigen.Factorize(a, true)
 	if !ok {
-		return matrix.ErrFailedEigen
+		return ErrFailedEigen
 	}
 	values := eigen.Values(nil)
 	for i, v := range values {
 		if v <= 0 {
-			return matrix.ErrNotPSD
+			return ErrNotPSD
```

Comment:
Unrelated to this PR but now that I see it: positive symmetric definite looks weird, positive belongs to definite. I would put symmetric first: symmetric positive definite (or SPD).